xradar/io/export/cfradial1.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python
+# Copyright (c) 2022, openradar developers.
+# Distributed under the MIT License. See LICENSE for more info.
+
+"""
+
+CfRadial1 output
+================
+
+This sub-module contains the writer for export of CfRadial1-based radar
+data.
+
+Author: @syedhamidali.
+
+Example::
+
+    import xradar as xd
+    dtree = xd.io.to_cfradial1(dtree, filename)
+
+.. autosummary::
+   :nosignatures:
+   :toctree: generated/
+
+   {}
+
+"""
+
+__all__ = [
+    "to_cfradial1",
+]
+
+import numpy as np
+import xarray as xr
+
+
+def _calib_mapper(calib_params):
+    """
+    Map calibration parameters to a new dataset format.
+
+    Parameters:
+    - calib_params: xarray.Dataset
+        Calibration parameters dataset.
+
+    Returns:
+    xarray.Dataset
+        New dataset with mapped calibration parameters.
+    """
+    new_data_vars = {}
+    for var in calib_params.data_vars:
+        data_array = calib_params[var]
+        new_data_vars["r_calib_" + var] = xr.DataArray(
+            data=data_array.data[np.newaxis, ...],
+            dims=["r_calib"] + list(data_array.dims),
+            coords={"r_calib": [0]},
+            attrs=data_array.attrs,
+        )
+    radar_calib_renamed = xr.Dataset(new_data_vars)
+    dummy_ds = radar_calib_renamed.rename_vars({"r_calib": "fake_coord"})
+    del dummy_ds["fake_coord"]
+    return dummy_ds
+
+
+def _main_info_mapper(dtree):
+    """
+    Map main radar information from a radar dtreeume dataset.
+
+    Parameters:
+    - dtree: datatree.Datatree
+        Radar datatree.
+
+    Returns:
+    xarray.Dataset
+        Dataset containing the mapped radar information.
+    """
+    dataset = (
+        dtree.root.to_dataset()
+        .drop("sweep_group_name")
+        .rename({"sweep_fixed_angle": "fixed_angle"})
+    )
+    return dataset
+
+
+def _variable_mapper(dtree, sweep_group_name):
+    """
+    Map radar variables for different sweep groups.
+
+    Parameters:
+    - dtree: datatree.Datatree
+        Radar datatree.
+    - sweep_group_name: xarray.DataArray
+        DataArray containing sweep group names in radar datatree.
+
+    Returns:
+    xarray.Dataset
+        Dataset containing mapped radar variables.
+    """
+    sweep_group_names = sweep_group_name.values.tolist()
+
+    sweep_datasets = []
+
+    for name in sweep_group_names:
+        data = dtree[name]
+
+        # Check if 'x', 'y', and 'z' variables are available before dropping them
+        if "x" in data.variables:
```

Comment:
You can remove the checks and use 

data = data.drop_vars(["x", "y", "z"], errors="ignore")